tests/introspection/tests.py

```diff
@@ -211,3 +211,130 @@ def test_get_constraints_indexes_orders(self):
                 self.assertEqual(val['orders'], ['ASC'] * len(val['columns']))
                 indexes_verified += 1
         self.assertEqual(indexes_verified, 4)
+
+    def test_get_constraints(self):
+        def assert_details(first, second):
+            # different backend introspection realizations has different data
+            first = {k: v for k, v in first.items()
```

Comment:
Use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
changed